src/dev/impl/DevToys/Models/JwtDecoderEncoder/DecoderParameters.cs

```diff
@@ -0,0 +1,20 @@
+﻿#nullable enable
+
+using DevToys;
+
+namespace DevToys.Models.JwtDecoderEncoder
+{
+    public class DecoderParameters
```

Comment:
Would it be better if it was a `struct`, or even a `record` (not sure if UWP supports it) ?